.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby:
+          - 2.4
+          - 2.5
+          - 2.6
+          - 2.7
+        gemfile:
+          - Gemfile
+          - gemfiles/activesupport-5.0.gemfile
+          - gemfiles/activesupport-5.1.gemfile
+          - gemfiles/activesupport-6.0.gemfile
+    name: Ruby ${{ matrix.ruby }} ${{ matrix.gemfile }}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Ruby ${{ matrix.ruby }}
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: Set up gemfile
+        run: |
+          export BUNDLE_GEMFILE=${GITHUB_WORKSPACE}/${{ matrix.gemfile }}
+          rm -f ${GITHUB_WORKSPACE}/Gemfile.lock
```

Comment:
This is before `bundle install` is ran, is there a `Gemfile.lock` there?

Reply:
No, it's not there now. And it's not necessary. I saw it in `shopify/shopify_api` and I like it as a defence against one getting committed accidentally. I can remove it.